### PR TITLE
Add git configuration to optionally fetch tags

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -578,10 +578,20 @@ Options available here are shared with `additional_repositories`.
 
 ```yaml
 git:
```
Comment (prusse-martin) [inline README.md:580]: You documented a lot of new options here, but only "tags" is explicit tested.
Is the rest already somehow tested?

**Review by prusse-martin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nicoddemus) [reply README.md:580]: Yes, supported and tested, just not documented for some reason.

Comment (prusse-martin) [reply README.md:580]: OK.

That the "key" types are validated just by the outer most type and no structure do not help us with this either =)

https://github.com/ESSS/jobs_done10/blob/master/src/jobs_done10/jobs_done_job.py#L73-L79

**Review by prusse-martin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nicoddemus) [reply README.md:580]: Yep, thought the same. 😖

Comment (prusse-martin) [reply README.md:580]: Maybe add that to documentation in "readme".

Comment (nicoddemus) [reply README.md:580]: What do you mean?

Comment (prusse-martin) [reply README.md:580]: ah... didn't notice this was on readme, my bad.

**Pull request merged**